public/docs/ts/latest/guide/glossary.jade

```diff
@@ -0,0 +1,445 @@
+    between a "token" or "key" and a dependency [provider](#provider).
+    This more rare usage should be clear in context.
+    
+:markdown
+  ## Bootstrap
+.l-sub-section
+  :markdown
+    We launch an Angular application by "bootstrapping" it with the `bootstrap` method.
+    The `bootstrap` method identifies the application's' top level "root" [Component](#component)
+    and optionally registers service [providers](#provider) with the 
+    [dependency injection system](#dependency-injection).
+
+        
+.l-main-section
+:markdown
+  ## Component
+.l-sub-section
+  :markdown
+    An Angular class responsible for exposing data 
+    to a [View](#view) and handling most of the view’s display 
+    and user-interaction logic. 
+    
+    The Component is one of the most important building blocks in the Angular system.
+    It is, in fact, an Angular [Directive](directive) with a companion [Template](#template).
+    
+    The  developer applies the `@Component` [decorator](decorator) to
+    the component class, thereby attaching to the class the essential component metadata 
+    that Angular needs to create a component instance and render it with its template
+    as a view.
+    
+    Those familiar with "MVC" and "MVVM" patterns will recognize
+    the Component in the role of "Controller" or "View Model".
+  
+.l-main-section
+:markdown
+  ## Data Binding
+.l-sub-section
+  :markdown
+    Applications display data values to a user and respond to user
+    actions (clicks, touches, keystrokes).
+    
+    We could push application data values into HTML, attach
+    event listeners, pull changed values from the screen, and
+    update application data values ... all by hand.
+    
+    Or we could declare the relationship between an HTML widget
+    and an application data source ... and let a data binding
+    framework handle the details.
+    
+    Data Binding is that second approach. Angular has a rich
+    data binding framework with a variety of data binding
+    operations and supporting declaration syntax.
+    
+     The many forms of binding include:
+     * [Interpolation](./template-syntax.html#interpolation)
+     * [Property Binding](./template-syntax.html#property-binding)
+     * [Event Binding](./template-syntax.html#event-binding)      
+     * [Attribute Binding](./template-syntax.html#aattribute-binding)     
+     * [Class Binding](./template-syntax.html#class-binding)     
+     * [Style Binding](./template-syntax.html#style-binding)     
+     * [Two-way data binding with ng-model](./template-syntax.html#ng-model)
+     
+    Learn more about data binding in the 
+    [Template Syntax](./template-syntax.html#data-binding) chapter.
+    
+:markdown
+  ## Decoration
+.l-sub-section
+  :markdown
+    A feature of TypeScript and ES2015.
+    
+    A Decoration is a function that adds Angular metadata to a class, 
+    constructor parameter, or a property.
```

Comment:
Maybe we should reword it as "a class, its members (properties, methods) and function arguments"

Since AFAIK it can apply to any argument, not only on constructors.